Makefile

```diff
@@ -533,38 +533,38 @@ test-wasm: $(WASM_MODULE_OUTPUTS)  ### Run engine tests
 
 .PHONY: bench
 bench: ## Run benchmarks in /benches
-	${MAYBE_ENVIRONMENT_EXEC} cargo bench --no-default-features --features "benches" ${CARGO_BENCH_FLAGS}
+	${MAYBE_ENVIRONMENT_EXEC} cargo bench --no-default-features --features "benches" ${CARGO_BENCH_FLAGS:-""}
```

Comment:
I'm curious to see if this works since `CARGO_BENCH_FLAGS` is currently defaulted to `""` at the head of this file.

I think you might need to remove the default instead like:

```diff
- export CARGO_BENCH_FLAGS ?= ""
+ export CARGO_BENCH_FLAGS ?=
```

Reply:
Good call.  I was thinking that there'd be no way to drop the default value, since like... why would you do assignment with no value.  That's what I get for assuming and not trying. :)

Fixed!